src/Moq/Mock.cs

```diff
@@ -395,36 +393,53 @@ internal void AddInnerMock(MethodInfo method, object returnValue)
 		{
 			Debug.Assert(Mock.TryGetFromReturnValue(returnValue, out _));
 
-			this.InnerMocks.TryAdd(method, returnValue);
+			var arguments = GetArguments(method);
+			var obj = Expression.Parameter(this.MockedType, "mock");
+			var expression = Expression.Lambda(Expression.Call(obj, method, arguments), obj);
+
+			this.Setups.Add(new FixedReturnValueSetup(method, arguments, expression, returnValue));
+
+			Expression[] GetArguments(MethodInfo m)
```

Comment:
This is were the difference between setups and `InnerMocks` entries becomes apparent: The former carry argument information, while the latter don't. As we refactor from `InnerMocks` towards setups, this lack of argument information becomes explicit, and we need to fill in the blanks with `It.IsAny<TArg>`. These are temporary placeholders; later we'll be able to build smarter setups.